apps/website/screens/components/data-grid/code/DataGridCodePage.tsx

```diff
@@ -262,14 +278,74 @@ const sections = [
       </DxcTable>
     ),
   },
-
+  ,
+  {
+    title: "DxcDataGrid.ActionsCell",
+    content: (
+      <DxcParagraph>
+        A compound component aimed to be used inside the table to display up to three actions.
+      </DxcParagraph>
+    ),
+    subSections: [
+      {
+        title: "Props",
+        content: (
+          <DxcTable>
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Type</th>
+                <th>Description</th>
+                <th>Default</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <DxcFlex direction="column" gap="0.25rem" alignItems="baseline">
+                    <StatusBadge status="required" />
+                    actions
+                  </DxcFlex>
+                </td>
+                <td>
+                  <ExtendedTableCode>{actionsType}</ExtendedTableCode>
+                </td>
+                <td>
+                  <p>
+                    It represents a list of interactuable elements that will work as buttons or as a dropdown. Those
```

Comment:
`interactuable` is not correct, please place `interactable` or `interactive` instead (I personally prefer the latter).

I've realised that this typo exists in other doc and also in the `Header` and `Chip` implementation. We should fix that.